publishexe.targets

```diff
@@ -38,6 +37,11 @@
   <Target Name="FixUpCoreConsoleNames">
     <Move SourceFiles="$(Outdir)$(AssemblyName).exe" DestinationFiles="$(Outdir)$(AssemblyName).dll" />
     <Move SourceFiles="$(Outdir)CoreConsole.exe" DestinationFiles="$(Outdir)$(AssemblyName).exe" />
+    <!-- create a marker indicating that the file was renamed, this will be used during signing to sign the correct file -->
+    <Touch Files="$(Outdir)$(AssemblyName).publishedexe" AlwaysCreate="true" />
+    <ItemGroup>
```

Comment:
OK perhaps we can update TargetPath or setup another property that could we could use for writing the signing file. I would prefer that we don't need to special cases things like this directly in GetFilesToSignItems and instead just have the extensibility point via some property into the target that creates the marker.